commands/update-ui/update-ui.sh

```diff
@@ -10,7 +10,12 @@ main() {
   # setup.
   cmd_runner_setup
 
-  rust_version="$1"
+  get_arg required --rust_version "$@"
+  rust_version="${out:-""}"
```

Comment:
If it's required, then we don't need to do `${out:-""}` and we can just do `$out`, right?
Same in other places.

Reply:
right, it's just easier format to copy & paste. the `required` check will just `exit 1` before assignment, so i'd prefer to keep it

Reply:
Ok, sounds good.